book/src/Drivers.md

```diff
@@ -9,8 +9,8 @@ Name | Status | More Information
 [NFS](https://github.com/kubernetes-csi/drivers/tree/master/pkg/nfs) | Sample | 
 [HostPath](https://github.com/kubernetes-csi/drivers/tree/master/pkg/hostpath) | Sample | Only use for a single node tests.
 [Flexvolume](https://github.com/kubernetes-csi/drivers/tree/master/pkg/flexadapter) | Sample |
-[Mock Driver](https://github.com/thecodeteam/gocsi/tree/master/mock) | Sample | Only use for development tests.
-
+[Mock Driver](https://github.com/thecodeteam/gocsi/tree/master/mock) | Released | A mock CSI plugin usable as a stand-alone binary or in code.
+[VFS Driver](https://github.com/thecodeteam/csi-vfs) | Released | A CSI plugin that provides a virtual file system.
```

Comment:
Could you add an empty line above the `##` for the header? If not then it will not interpret the header.